.pre-commit-config.yaml

```
@@ -44,7 +44,7 @@ repos:
       - id: python-check-mock-methods
 
   - repo: https://github.com/psf/black
-    rev: 21.9b0
+    rev: "22.3.0"
```

Hello, and thanks for your contribution.

Out of curiosity. Did you manually update this line?
```suggestion
    rev: 22.3.0
```

We usually run `pre-commit autoupdate`.

---

Yes, Id took a look at the latest version of this dep and updated it manually to keep the changes on this PR at minimal.
If I did pre-commit autoupdate I would have to fix more source code (I did not tried). So to keep the changes at minimal on this PR I just fixed what was broken.

If required, I can run pre-commit autoupdate as well.